src/world/partner/kooper.c

```diff
@@ -8,7 +8,16 @@ INCLUDE_ASM(s32, "world/partner/kooper", func_802BD144_31B164);
 
 INCLUDE_ASM(s32, "world/partner/kooper", func_802BD17C_31B19C);
 
-INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
+//INCLUDE_ASM(s32, "world/partner/kooper", func_802BD200_31B220);
+
+extern s32 D_802BEC54;
```

Comment:
Externs are typically put at the top of the file with eachother. This one would go before the unkPartnerStruct* one at the very top, to put them in ram order.